docs/copilot/ai-powered-suggestions.md

```diff
@@ -0,0 +1,208 @@
+---
+Order: 
+Area: copilot
+TOCTitle: AI-powered suggestions
+ContentId: 7ab2cd6c-45fd-4278-a6e8-1c9e060593ea
+PageTitle: AI-powered suggestions with GitHub Copilot
+DateApproved: 02/1/2024
+MetaDescription: Enhance your coding with AI-powered suggestions from GitHub Copilot in Visual Studio Code.
+---
+# GitHub Copilot in VS Code
+
+The [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension is an AI pair programmer tool that helps you write code faster and smarter. You can use the Copilot extension in Visual Studio Code to generate code, learn from the code it generates, and even configure your editor.
+
+With GitHub Copilot in VS Code you can:
+
+* Get inline code suggestions while you're writing and iterating on code.
+* Start a chat conversation to generate or refactor source code, produce documentation comments, or generate unit tests.
+* Ask questions to help ramp-up on a new code base, or accelerate learning a new programming language or framework.
+* Use chat features to discover and configure your VS Code setup.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Fi3AJZZregI" title="Get Started with the Future of Coding: GitHub Copilot" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Getting started
+
+If you want to use GitHub Copilot, you either need a subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization.
+
+The following tutorials walk you through using GitHub Copilot in VS Code.
+
+1. [Getting started](/docs/copilot/getting-started.md): set up VS Code for Copilot and get your first AI-powered suggestions
+
+1. [Getting started with Copilot Chat](/docs/copilot/getting-started-chat.md): install the Copilot Chat extension and have your first AI chat conversations
+
+## Using Copilot
+
+Now that you've signed up for Copilot and activated the extension, let's see its assistance in action!
+
+GitHub Copilot provides suggestions for numerous languages and a wide variety of frameworks, and it works especially well for Python, JavaScript, TypeScript, Ruby, Go, C# and C++.
```

Comment:
Nit: Python, JavaScript, TypeScript, Ruby, Go, C# and C++.

This language ordering is weird. Ruby being very high.
I suggest to just use top languages in VS Code.

We probably also have telemetry that shows top copilot languages

fyi @bamurtaugh 

Feel free to publish, and the later we can revisit.

Reply:
I kept this fragment from the original article. Makes sense, especially as this list might evolve over time.